src/gui/transferlistfilterswidget.h

```diff
@@ -131,7 +131,8 @@ class TrackerFiltersList final : public BaseFilterWidget
     void removeTrackers(const BitTorrent::Torrent *torrent, const QStringList &trackers);
     void refreshTrackers(const BitTorrent::Torrent *torrent);
     void changeTrackerless(const BitTorrent::Torrent *torrent, bool trackerless);
-    void handleTrackerEntriesUpdated(const QHash<BitTorrent::Torrent *, QSet<QString>> &updateInfos);
+    void handleTrackerEntriesUpdated(BitTorrent::Torrent *torrent
```

Comment:
`const BitTorrent::Torrent *torrent` or if pedantic `const BitTorrent::Torrent * const torrent`

Reply:
> `const BitTorrent::Torrent *torrent` or if pedantic `const BitTorrent::Torrent * const torrent`

Done (but not so pedantic).